### ISSUE TITLE
Modularize codebase

### ISSUE THREAD

**Issue by Thanus-Kumaar** [Thanus-Kumaar/FC-TeamForge-server#1]
### Required
The project structure is very simple with only a single server.js file containing all the necessary routes. Modularize the code into routes, controllers, modules, and other necessary structures like utils, middleware, etc.

- [ ] Create modules
- [ ] Create routes
- [ ] Create controllers
- [ ] Other directory structures

##### Please go ahead and create other necessary files that can reduce the duplication of code and increase the readability of the codebase. Suggestions are welcome

**Comment by priyansh-narang2308** [comment]
Hey @Thanus-Kumaar Can I try this ?

**Comment by Thanus-Kumaar** [comment]
Yeah sure @priyansh-narang2308 .

**Comment by Thanus-Kumaar** [comment]
> So I have to edit the server.js and divide it into diff component's like modules, routes... 
> Right

Yes

**Comment by priyansh-narang2308** [comment]
Hey @Thanus-Kumaar  I am creating a .env file as well so the inputs should i give mine own of sql
![image](https://github.com/user-attachments/assets/b5d54e91-61da-4186-8643-ae3c0d612510)

**Comment by priyansh-narang2308** [comment]
Yeahh I updated by my own details
![image](https://github.com/user-attachments/assets/4d990ec6-c7a4-4817-9f68-2982a2210c9a)

**Comment by priyansh-narang2308** [comment]
Hello @Thanus-Kumaar I have completed segregating the files and putting it in diff folders as
![image](https://github.com/user-attachments/assets/a161901d-2fa8-4e1a-94aa-22c4e7c7c1af)
Like this  and its working clean should i make a PR?
The output 
![image](https://github.com/user-attachments/assets/fbc89c61-92df-4200-9afa-c62fc252e1e5)

**Comment by priyansh-narang2308** [comment]
**I even attempted to put the utils folder which consisted a JS file to validation purposes like if a person doesnt enter a player name correctly and numbers are there it wud show error
But as soon as I run the server it shows an unexpected error..**

**Comment by Thanus-Kumaar** [comment]
@priyansh-narang2308 no problem..i will look into the utils part..but for now, what you have done is more than enough..create a PR, and i will review it.

**Comment by Thanus-Kumaar** [comment]
> Yeahh I updated by my own details
> ![image](https://github.com/user-attachments/assets/4d990ec6-c7a4-4817-9f68-2982a2210c9a)
> 
Bro 😑, you should not show these ones directly like this..

**Comment by priyansh-narang2308** [comment]
Ohh sorryyy @Thanus-Kumaar 
Should I delete it from my code and then make a PR 
Though the server won't run

**Comment by priyansh-narang2308** [comment]
Hey @Thanus-Kumaar I have made the PR pls do review🫡🫡

**Comment by Thanus-Kumaar** [comment]
> Ohh sorryyy @Thanus-Kumaar Should I delete it from my code and then make a PR Though the server won't run

No, that's not what I meant, you should not share your creds like this openly on a public platform...You should only share the key not the value..

**Comment by priyansh-narang2308** [comment]
Ohh okk @Thanus-Kumaar but the Pr that I made doesn't include these cred so should I edit the pr?

**Comment by Thanus-Kumaar** [comment]
> Ohh okk @Thanus-Kumaar but the Pr that I made doesn't include these cred so should I edit the pr?

No, your PR should not include those..dont that any env variables there..i will review and suggest other changes if necessary

**Comment by priyansh-narang2308** [comment]
Yeahh ok @Thanus-Kumaar 👍🏻👍🏻
